.travis.yml

```diff
@@ -16,3 +20,9 @@ notifications:
     on_success: never
     on_failure: always
     on_start: never
+addons:
```

Comment:
Theses settings will be described in the release process part (I don't think here is the good place to do it)